processor/src/main/java/org/mapstruct/ap/internal/model/source/Mapping.java

```diff
@@ -166,7 +189,8 @@ public static Mapping forIgnore( String targetName) {
             null,
             null,
             null,
-            new ArrayList(),
+            new HashSet<>(),
```

Comment:
Since we are changing I think it is better to use `Collections.emptySet()`. Like in `fromMappingPrism`

Reply:
Agreed